lib/MusicBrainz/Server/Controller/WS/2/Release.pm

```diff
@@ -153,7 +153,9 @@ sub release_toplevel {
         # number of recordings (audiobooks) will almost always timeout, driving up
         # server load average for nothing. Ideally we can remove this limit as soon as
         # we resolve the performance issues we have.
-        my $max_recording_relationships = 500;
+        my $max_recording_relationships = 500 * (
+            $stash->_data->{_json_dump} ? 'inf' : 1
```

Comment:
Right.  There's no request timeout at all for the dumps, because the dump process isn't actually making requests: it calls directly into the controller methods that build the JSON (see lib/MusicBrainz/Server/JSONLookup.pm).  So we can just disable this limitation entirely.  (The most recent full dump ran with this change and it was fine, though I'm guessing it took slightly longer than before.)

The `natatime` change was just a precaution.  Since previously we were only fetching relationships for up to 500 recordings per batch, and we know that hasn't caused issues, I divided the relationship loading by that constant.  I didn't want to worry about overloading PG with long-running queries that affect the performance of other processes.